ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,243 @@
+by making all the benefits of Account Abstraction a core part of the protocol, bringing down the cost for the users,
+enabling easy migration to post-quantum cryptography and eventually deprecation of Externally Owned Accounts altogether.
+
+There are also some actions that are only applicable in the context of Native Account Abstraction,
+which requires a creation of new JSON-RPC API methods.
+
+## Specification
+
+This document defines the API that the RIP-7560 compatible Ethereum nodes provide to either Smart Contract
+Wallet applications or advanced dapps.
+We define the following changes to the Ethereum JSON-RPC API:
+
+### Definition of the RIP-7560 transaction type:
+
+The following table represents a full list of fields of an RIP-7560 transaction:
+
+| Name                          | Type           | Description                                                    |
+|-------------------------------|----------------|----------------------------------------------------------------|
+| sender                        | DATA, 20 Bytes | Address of the Smart Contract Account making the transaction   |
+| deployer                      | DATA, 20 Bytes | Address of the Deployer - account factory contract             |
+| deployerData                  | DATA           | Data that is provided to the Deployer contract                 |
+| paymaster                     | DATA, 20 Bytes | Address of the Paymaster contract                              |
+| paymasterData                 | DATA           | Data that is provided to the Paymaster contract                |
+| callData                      | DATA           | Data that is provided to the Account contract for execution    |
+| nonce                         | QUANTITY       | A 256 bit nonce. Use of `nonce > 2^64` is defined in RIP-7712  |
+| builderFee                    | QUANTITY       | Value passed from sender or paymaster to the `coinbase`        |
+| maxPriorityFeePerGas          | QUANTITY       | The maximum gas price to be included as a tip to the validator |
+| maxFeePerGas                  | QUANTITY       | The maximum fee per unit of gas                                |
+| validationGasLimit            | QUANTITY       | Gas provided for the transaction account validation frame      |
+| paymasterValidationGasLimit   | QUANTITY       | Gas provided for the transaction paymaster validation frame    |
+| paymasterPostOpGasLimit       | QUANTITY       | Gas provided for the transaction paymaster `postOp` frame      |
+| callGasLimit                  | QUANTITY       | Gas provided for the transaction execution frame               |
+| accessList                    | OBJECT         | An EIP-2930 compatible Access List structure                   |
+| EIP-7702 authorizations (WIP) | ARRAY          | An EIP-7702 compatible list of contracts injected into EOAs    |
+| signature                     | DATA           | A signature of any kind used by Account to verify transaction  |
```

Comment:
If we're renaming this to `authData`, remember to do it here as well.  Ditto for `callData` above.